github/repos_environments.go

```diff
@@ -17,6 +17,7 @@ type Environment struct {
 	Owner                  *string         `json:"owner,omitempty"`
 	Repo                   *string         `json:"repo,omitempty"`
 	EnvironmentName        *string         `json:"environment_name,omitempty"`
+	PreventSelfReview      *bool           `json:"prevent_self_review,omitempty"`
```

Comment:
Why is this needed in the `Environment` struct?  
Using the link in your PR description, I only see it being needed in the `ProtectionRule`.

Reply:
Good point, I thought this was the same struct used for CreateUpdateEnvironment.